### PR TITLE
fix sign bug

### PR DESCRIPTION
**Pull request by duanyutong**
- [x] fix sign bug

### CODE REVIEW

File: toppra/dracula/__init__.py [hunk 1]
```diff
@@ -102,7 +102,9 @@ def RunTopp(
     # toppra only produces clamped cubic splines
     path = ta.SplineInterpolator(x, waypts.copy(), bc_type="clamped")
     # avoid going over limit taking into account toppra's precision
-    pc_vel = constraint.JointVelocityConstraint(vlim - JOINT_DIST_EPS)
+    pc_vel = constraint.JointVelocityConstraint(
+        vlim - np.sign(vlim) * JOINT_DIST_EPS
```
Comment (dmsj) [inline toppra/dracula/__init__.py]: `JOINT_DIST_EPS` has different units than the desired `VELOCITY_EPS`, even if they are nearly the same numerical value. I would expect them to change differently depending on the type of trajectory.

**Review by dmsj — COMMENTED** [review]
propose adding another variable

File: toppra/dracula/__init__.py (hunk 1 above)
Comment (duanyutong) [reply toppra/dracula/__init__.py]: will add another variable when we do find that the velocity eps diverges from the truncation epsilon

**Comment by duanyutong** [comment]
merging after testing. remaining issues lie within our C++ wrapper.

**Pull request merged**